src/admin/components/forms/field-types/Checkbox/Input.tsx

```diff
@@ -43,9 +46,12 @@ export const CheckboxInput: React.FC<CheckboxInputProps> = (props) => {
         name={name}
         checked={checked}
         readOnly
+        tabIndex={-1}
```

Comment:
I believe you should just be able to use [aria-disabled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled). Will need to test and verify.